autospec/files.py

```diff
@@ -290,7 +309,6 @@ def push_file(self, filename):
             (r"^/etc/systemd/system/.*\.wants/", "active-units"),
             # now a set of catch-all rules
             (r"^/etc/", "config", "", "%config "),
```

Comment:
Should these two patterns beginning with `/etc` be removed as well?

Reply:
Yes they should, thanks!